component/administrator/language/en-GB/com_jcomments.ini

```
@@ -255,8 +255,8 @@ AP_COMMENT_MINLENGTH="Minimum comment length"
 AP_COMMENT_MINLENGTH_DESC="The minimum number of characters a user should post. The user will get an alert if comment's length less then this value."
 AP_SHOW_COMMENTLENGTH="Character counter"
 AP_SHOW_COMMENTLENGTH_DESC="Enable/Disable displaying the character count while typing."
-AP_ENABLE_ACCEPT_TERMS_OF_USE="Don't show checkbox \"Accept terms of use\"."
-AP_ENABLE_ACCEPT_TERMS_OF_USE_DESC="Enable/Disable displayint the checkbox to accept terms of use."
+AP_ENABLE_ACCEPT_TERMS_OF_USE="Show the \"Accept terms of use\"."
```

Minus the dot, plus checkbox:

```suggestion
AP_ENABLE_ACCEPT_TERMS_OF_USE="Show the \"Accept terms of use\" checkbox"
```